internal/renderers/skia/Cargo.toml

```diff
@@ -52,6 +52,8 @@ winit = { version = "0.28.6", optional = true, default-features = false }
 ash = { version = "^0.37.2", optional = true }
 vulkano = { version = "0.33.0", optional = true, default-features = false }
 
+softbuffer = { workspace = true }
```

Comment:
I wouldn't make it optional, but one thing that I should have done and will do now is to limit the features we select on softbuffer, so that a wayland build doesn't pull in x11, and we don't pull in kms for now.

Reply:
On second thought, I'll make a separate PR for that :)